docs/content/Modpacks/Recipes/Recipe-Conditions.md

```diff
@@ -2,151 +2,48 @@
 title: Recipe Conditions
 ---
 
-!!! Warning 
-    Custom recipe conditions are only supported in Java. Therefore, this page will only contain Java examples.
+Recipe Conditions are recipe properties that can prevent a recipe from starting based on certain criteria, like for example Biome, Weather, Quest Completions, or self-made custom Conditions. These conditions can be very helpful in spicing up your machines and systems!
 
-Recipe conditions are custom conditions for your recipe, like biome, machine tier, or anything else you can think of.
+These conditions can be used in both java and kubejs recipes. However, custom conditons can only be done in java. If you want to see how to make these, check out the [Custom Recipe Condition](./Custom Recipe Condition.md) example page.              
```

Comment:
WARNING -  Doc file 'Modpacks/Recipes/Recipe-Conditions.md' contains a link './Custom Recipe Condition.md', but the target 'Modpacks/Recipes/Custom Recipe Condition.md' is not found among documentation files.

Change this to the right page (I think ../Custom-Recipe-Condition.md)